index.html

```diff
@@ -166,4 +168,32 @@ <h1>About <br> Myself</h1>
     </div>
   </section>
 </body>
-</html>
\ No newline at end of file
+
+<section class="formsection">
+  <div class="heading-4">
+    <h2>Contact me</h2>
+  </div>
+  <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it</p>
+  
+  <form action="https://formspree.io/f/mjvzjjpb" method="post"></form>
+    <div class="contact-box">
+      <div class="fullname">
+        <label for="name"></label>
+        <input type="text" id="name" name="user-name" maxlength="30" placeholder="enter full name" required>
+      </div>
+      <div class="email">
+        <label for="email"></label>
+        <input type="email" id="mail" name="user_email" placeholder="enter email" required>
+      </div>
+      <div class="message">
+        <label for="msg"></label>
+        <textarea id="msg" name="user_message" maxlength="500" placeholder="Write your message here" required></textarea>
+        <button class="btn">Get in touch</button>
+      </div>
+    </div>
+  </form>
```

Comment:
**OPTIONAL**- When I click on the submit button, the input text doesn't submit, this is because you have wrapped the form button inappropriately in the `form` tag. Please remove the closing `</form>` tag on line `178` 🙏 
**OPTIONAL** - Good job fixing the width of the contact section 👏 .Your contact section is missing a background image as shown in the design, please include this background image 👍 👍 .(I missed it in my first review, my apologies).

Figma Design:
<img width="194" alt="ffff" src="https://user-images.githubusercontent.com/73607512/201113036-790cf9b2-49c3-41f9-99c5-2a1757c75f8b.PNG">